git_build_branch/branch_builder.py

```diff
@@ -233,6 +231,25 @@ def rebuild_staging(config, path, print_details=True, push=True):
         exit(1)
 
 
+def print_not_found(cwd, branch, config):
+    print(red("  [{cwd}] {branch}".format(
+        cwd=format_cwd(cwd),
+        branch=branch,
+    )))
```

Comment:
nit: since you used f-strings below
```suggestion
    print(red(f"  [{format_cwd(cwd)}] {branch}"))
```